libraries/psibase/native/src/NativeFunctions.cpp

```diff
@@ -1,4 +1,5 @@
-#include <psibase/NativeFunctions.hpp>
+#include <psibase/RawNativeFunctions.hpp>
```

Comment:
There's no reason to `#include` this here because the host functions are not available in this form for native code.